### PR TITLE
my new code for review using PR

### PR DESCRIPTION
**Pull request by mohansgithub**
Hello Elango and Raj please review my code

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -11,9 +11,9 @@
 			<h1> Mohanraj Testing the commit message twice</h1>
 			<h1> Loka Mahesh DEVOPS Rocks Devops</h1>
 			<h1> Raj Canada Devops Devops DEVOPS</h1>			
-			<h1> Elango Bangalore Excellent DEVOPS</h1>
+			<h1> My code for review Elango Bangalore Excellent DEVOPS</h1>
```
Comment (selangovan400) [inline src/main/webapp/WEB-INF/views/home.jsp:14]: ok. good

Comment (Hemachandra-423) [inline src/main/webapp/WEB-INF/views/home.jsp:14]: good one

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 2]
```diff
@@ -11,9 +11,9 @@
...
 			<h1> vijay kumar</h1>
-			<h1> MADHES DEVOPS</h1>
+			<h1> DevOps is automation of Dev and OpsMADHES DEVOPS</h1>
```
Comment (selangovan400) [inline src/main/webapp/WEB-INF/views/home.jsp]: please add todays date.

Comment (Hemachandra-423) [inline src/main/webapp/WEB-INF/views/home.jsp:16]: date is missing

**Review by Hemachandra-423 — DISMISSED** [review]
please fix this asap

**Review by selangovan400 — CHANGES_REQUESTED** [review]
please fix

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 1 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:14]: thanks

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 2 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp]: Added as requested

**Review by mohansgithub — COMMENTED** [review]
Updated as per comment

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 2 above)
Comment (selangovan400) [reply src/main/webapp/WEB-INF/views/home.jsp]: good.

**Review by selangovan400 — APPROVED** [review]

**Comment by mohansgithub** [comment]
Merging it is approved

**Pull request merged**